tools/check-license.py

```diff
@@ -46,7 +46,8 @@
 ]
 
 exclude_dirs = [
-    'targets/esp8266'
+    'targets/esp8266',
+    'tests/test262',
```

Comment:
It would be better to use `TEST262_TEST_SUITE_DIR` from settings.py, but it is an absolute path. Should be get a relative path from PROJECT_DIR and TEST262_TEST_SUITE_DIR. @galpeter, can we do it easily in python?